cmd/scollector/service_windows.go

```diff
@@ -71,7 +73,9 @@ func installService(name, desc string) error {
 		return err
 	}
 	defer s.Close()
-	err = eventlog.InstallAsEventCreate(name, eventlog.Error|eventlog.Warning|eventlog.Info)
+	// Workaround for bug in eventlog.InstallAsEventCreate
```

Comment:
This workaround is no longer needed, as I included the upstream patch. We still need to fix any existing installs that are currently broken by manually expanding the %systemroot% variable in the REG_SZ key. Ping me when you have time to jump on a hangout if you want more details.